packages/itwin/imodel-react-hooks/package.json

```diff
@@ -1,34 +1,38 @@
 {
-  "name": "@bentley/imodel-react-hooks",
-  "version": "0.1.4",
-  "description": "Reusable React Hooks to help with iModel.js",
+  "name": "@itwin/imodel-react-hooks",
+  "version": "0.1.0",
+  "description": "Reusable React Hooks to help with iTwin.js",
   "keywords": [
     "Bentley",
     "hooks",
     "react",
-    "iModel.js"
+    "iModel",
+    "iTwin.js"
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/imodel-react-hooks"
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/itwin/imodel-react-hooks"
```

Comment:
@calebmshafer  @skirby1996 thoughts on pkg name? itwin/imodel-react-hooks?

Reply:
`imodel-react-hooks` seems fine with me. It'd be a bit awkward to use our standard pattern of putting `-react` as the suffix since the hooks are react specific. Plus it reads weird as `@itwin/imodel-hooks-react`.

As for whether to drop the imodel, I have no problem with that if we are going to expand upon the list of hooks. Do they cover anything beyound Marker and FeatureOverrides at the moment?

Reply:
theyre hook based wrappers over functionality from core-frontend.

we do have the [bentley/react-hooks](https://github.com/iTwin/react-hooks) pkg too for more generic hooks

Reply:
There is definitely room for more hooks and react wrapping. I'd hope someone could contribute if they have or want them. That is the intended scope of this package.

Reply:
itwin/imodel-react-hooks is what we're going to go with; resolving